packages/fluent-theme/src/components/assets/SlidingDots.tsx

```diff
@@ -0,0 +1,62 @@
+const SLIDING_DOTS_SVG_URL = URL.createObjectURL(new Blob([SLIDING_DOTS_SVG_STRING], { type: 'image/svg+xml' }));
```

Comment:
Should we have an object URL composer? I'm ok without it, but I have to ask.

Reply:
I was thinking the same too. Let's keep it here and we should be doing that one soon.